Sources/CardPayments/APIRequests/UpdateSetupTokenRequest.swift

```diff
@@ -0,0 +1,81 @@
+import Foundation
+import CorePayments
+
+class UpdateSetupTokenQuery: Codable, GraphQLQuery {
```

Comment:
It looks like we have to classes named `UpdateSetupTokenQuery`. Is this a typo?

Reply:
It's something weird that happened with Xcode I can't figure out. I don't see the query file in the workspace project navigator but I see it in finder. I had some weird issues with this earlier. I will check if they're exact duplicates and delete one of them in the finder.

Reply:
Resolved in 9025e95a60278ab97224737eea0aac7d9847a780 and 015ff93f14f56cc716d068c0e27564bfc4532264